scripts/db_migrations/v4.10/cleanIssues.py

```diff
@@ -0,0 +1,48 @@
+import sys
+from pymongo import MongoClient
+
+if len(sys.argv) < 5:
+    print("Not enough arguments.")
+    print("cleanIssues.py <mongoURL> <mongoPort> <userName> <password>")
+    sys.exit(0)
+
+mongoURL = sys.argv[1]
+mongoPort = sys.argv[2]
+userName = sys.argv[3]
+password = sys.argv[4]
+
+connString = "mongodb://"+ userName + ":" + password +"@"+mongoURL + ":" + mongoPort + "/"
+
+##### Enable dry run to not commit to the database #####
+dryRun = True
+
+##### Connect to the Database #####
+db = MongoClient(connString)
+for database in db.database_names():
+    if database != "admin" and database != "local":
+        db = MongoClient(connString)[database]
+        print("--database:" + database)
+
+##### Get a model ID and find entries #####
+        for setting in db.settings.find(no_cursor_timeout=True):
+            modelId = setting.get("_id")
+            print("\t--model: " +  modelId)
+            for issue in db[modelId + ".issues"].find({"$or":[{"comments._id":{"$exists":True}},{"viewpoints._id":{"$exists":True}},{"viewpoint":{"$exists":True}}]}):
+                issueId = issue.get("_id")
+                unsetFields = { "$unset": {} }
+                print("\t\t--issue: " +  str(issueId))
+##### Remove legacy viewpoint #####
+                if "viewpoint" in issue:
+                    unsetFields["$unset"]["viewpoint"] = ""
```

Comment:
We're removing potentially valid viewpoints here. 2 options:
1. check the contents of the viewpoint, we only want to remove the viewpoint if it's empty (the toy project situation)
2. if there's a valid viewpoint, move it to the first entry of viewpoints.